atom/browser/net/preconnect_manager_tab_helper.h

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2019 GitHub, Inc.
+// Use of this source code is governed by the MIT license that can be
+// found in the LICENSE file.
+
+#ifndef ATOM_BROWSER_NET_PRECONNECT_MANAGER_TAB_HELPER_H_
+#define ATOM_BROWSER_NET_PRECONNECT_MANAGER_TAB_HELPER_H_
+
+#include "content/public/browser/navigation_handle.h"
+#include "content/public/browser/web_contents_observer.h"
+#include "content/public/browser/web_contents_user_data.h"
+
+namespace predictors {
+class PreconnectManager;
+}
+
+namespace atom {
+
+class WebContentsPreferences;
+
+class PreconnectManagerTabHelper
```

Comment:
While the relevant class in Chrome is called a `TabHelper`, Electron doesn't have the concept of "tabs" so I think this would be better named e.g. `PreconnectManagerHelper`

Reply:
fixed